### ISSUE TITLE
Labels of MultiSelectCard's don't update when changed

### ISSUE THREAD

**Issue by spig0t** [CodeFoxLk/flutter_multi_select#5]
I'm changing the label for a MultiSelectCard within a setState call within the MultiSelectContainer's onChange call. When I debug, I can see that build is being called, and a new MultiSelectContainer is being created, with the relevant MultiSelectCard being given the new value for its value, yet the UI doesn't change and still shows the old value (and presumably is still somehow showing the old MultiSelectCard even though I can see a new one was created with the correct new value).

**Comment by spig0t** [comment]
Ah, just to be clear - the _label_ for the MultiSelectCard is what is changed, and while build runs and the whole MultiSelectContainer and all MultiSelectCard's are re-created, somehow the old label still shows.

**Comment by spig0t** [comment]
Setting the MultiSelectContainer's key to a new value each time something is changed seems to workaround this issue, but it's still not clear why this is necessary to me ...?

**Comment by CodeFoxLk** [comment]
yes. MultiSelectContainer does not recreate itself. you can add a key like this. 
```
MultiSelectContainer(
               key: UniqueKey(),
```

**Comment by CodeFoxLk** [comment]
have you tested this with the newest version?

**Comment by AG-Akash09** [comment]
In the latest version still this issue is present.
